dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServlet.java

```diff
@@ -144,7 +149,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             return;
         }
 
-        if (OidcClientUtil.isReferrerHostValid(request, requestUrl) == false) {
+        if (isRedirectionUrlValid(request, requestUrl) == false) {
```

Comment:
Previously, this `if` block did not exist so there would never be any kind of error or exception. Now, if not beta (i.e. GA), you will get a UnsupportedOperationException. I think you need to change this beta guard to simply skip this block if not in beta.
```
if (ProductInfo.getBetaEdition()) {
   if (isRedirectionUrlValid(request, requestUrl) == false) {
   ...
   }
}
```

Reply:
@tevans78  Pushed new commit to skip the check, but still issue the beta message on first call.